scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -2,7 +2,11 @@
 function populateCytobands(cytobands){
   var chromPosPattern = new RegExp("^(?:chr)?([1-9]|1[0-9]|2[0-2]|X|Y|MT)$");
   var chrom = document.forms["filters_form"].elements["chrom"].value;
-  var chromPos = document.forms["filters_form"].elements["chrom_pos"].value;
+  var chromPos = "";
+  if (typeof document.forms["filters_form"].elements["chrom_pos"] != "undefined") {
```

Comment:
```suggestion
  if (typeof document.forms["filters_form"].elements["chrom_pos"] !== "undefined") {
```

Reply:
Nice that you found the problem! You probably tried this solution and it works, so feel free to ignore my suggestion. But I was thinking that if you are doing a type comparison perhaps the triple equals could be more appropriate than then double equal way (check the docs [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) for a description of the differences)

Reply:
Thanks, yes, that appears correct, though there is no real issue with using the double equal here either - typeof will return a string regardless of what came in.